sec2-architecture.tex

```diff
@@ -155,91 +188,137 @@ \subsubsection{LSST-specific Objects}\label{lsst-specific-objects}
+possible for a user at one DAC to get access to their workspace from another
+(presuming that they have access rights at both DACs in the first place),
+albeit perhaps at reduced performance compared to the DAC-local workspace.
+
+All users will have a basic quota of User File Workspace storage, and may then
+apply to a resource allocation committee for additional storage for specific
+purposes.
 
 User file workspaces are backed up for disaster recovery purposes, but they are
 not fully protected against user error.
 
 \subsubsection{Data Access Permissions and Quotas}\label{data-access-permissions-and-quotas}
 
-All LSST data products are available to all users with LSST data rights.  Users
-who had LSST data rights but have since lost them may be granted access to the
-data products available as of the time of loss but not newer ones.
+All LSST data products are available to all users with LSST data rights.
+Users may grant other users or groups of users access to data in their User
+File and/or Database Workspaces.
+Such access may be read-only or read-write.
 
-Users may grant other users or groups of users access to catalogs and files in
-their workspace.  Such access may be read-only or read-write.
+The system is designed with the flexibility to permit users who had LSST data
+rights but have since lost them (e.g.,\ due to a change in country of
+employment) to continue to be granted access to Data Release data products
+available as of the time of loss, but not newer ones.
+However, the actual policy in this are will be determined by the Operations
+organization.
 
 Per-user catalogs and files are a limited resource with quotas determined by
-resource management (see section~\ref{resource-management}).  Some "user"
-catalogs and files may be considered to be owned by a group, with a distinct
-quota.
+resource management (see section~\ref{resource-management}).
+Some ``user'' catalogs and files may be considered to be owned by a group,
+with a distinct quota.
 
 \subsubsection{Support of Previous Releases}\label{support-of-previous-releases}
 
 The LSST baseline currently anticipates that the data products contained in the
-last two Data Releases (as well as Level 1 Alert Production data products) will
-be available through the LSST Science Platform.  Data products from other Data
-Releases would need to be retrieved via the Bulk Distribution service; they
-would not be visible within the LSP.
+last two Data Releases, as well as the Prompt / Level 1 data products from
+Alert Production) will be available through the LSST Science Platform.
+Data products from previous Data Releases would need to be retrieved via the
+Bulk Distribution service; they would not be visible within the LSP.
 
 A request to have the catalogs from all Data Releases be available through
-the LSP is being evaluated.
+the LSP is currently being evaluated through the LSST Change Control process.
```

Comment:
Which LCR is that?

Reply:
Good catch.  That was LCR-908, but of course since the LDM-542 text was written that LCR was adopted and implemented.  I'll edit accordingly.